pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -68,10 +69,30 @@ If you are presented with a 'recovery login' prompt this usually indicates an is
 * The SD card itself is faulty - older SD cards, especially ones which have been used a lot and thus may also be *physically* worn at the pins can be unreliable, resulting in data corruption. Try using a new SD card.
 
 ### Wifi connection problems on RPI3
+
 If you are having issues with your RPI3 connecting to your wifi, make sure
 to check that the 2.4 GHz channel on your wifi router is set to something less than 11, since
 the firmware on the Raspberry Pi doesn't support channel 12 and 13.
 
+### balenaOS boot issues with new hardware revision of RPi4
+
+The Raspberry Pi Foundation recently released [hardware Rev 1.5](https://forums.raspberrypi.com/viewtopic.php?t=329299) of the Raspberry Pi 4 Model B. This revision of the board requires a relatively recent version of [balenaOS](https://balena.io/os). We recommend using balenaOS v2.88.5+rev1 or later containing the updated bootloader and firmware versions. BalenaOS v2.75.0+rev1 and earlier versions aren't tested and possibly won't boot on this revision of the board.
```

Comment:
It wasn't the [Raspberry Pi Foundation](https://www.raspberrypi.org/), it was [Raspberry Pi](https://www.raspberrypi.com/). Raspberry Pi Ltd is the trading arm, we're the folks that produce the hardware, and maintain the software, and we're legally an entirely separate entity to the Foundation. There is no overlap in staffing or resources. We're both standalone entities.

Reply:
Thanks @aallan for catching this 👀 Appreciate you coming forward to rectify this. I will keep this in mind for the future and also inform the team of the same.

Reply:
Resolving..